### PR TITLE
Update docker image base to alpine:3.15

### PR DESCRIPTION
**Pull request by mdeggies**
This is required, as alpine:3.13 has a number of CVE's (which are all fixed in 3.15)!

### CODE REVIEW

**Review by claire-labry — APPROVED** [review]
👍🏻

**Comment by mdeggies** [comment]
Going to merge this in to unblock the pipeline 🙏

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/576871.

**Review by eculver — COMMENTED** [review]
(approved)

**Pull request merged**
